src/Boards/Board.php

```diff
@@ -140,10 +91,24 @@ public function compare(Board $_board)
      */
     public function setCell($_x, $_y, $_value)

```

Comment:
Is this method really necessary? Wouldn't be ok if users of the board would use `$board->field(1,3)->setValue(true);`?
If you really need it I would suggest renaming it to setFieldValue() to make it more clear what it really does.

Reply:
i've not thought about doing it this way thanks.

Reply:
now the problem is out of bounds check